pages/now-showing.js

```diff
@@ -10,13 +12,20 @@ const socketURL =
 const socket = io(socketURL);
 
 export default function NowShowing() {
+  const dispatch = useDispatch();
+  const messages = useSelector((state) => state.chat.messages);
+
   useEffect(() => {
```

Comment:
We actually don't really need this useEffect hook and it's probably confusing to leave it here. Will remove.